src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -140,7 +154,15 @@ static bool templateNotify
     std::string key   = it->first;
     std::string value = it->second;
 
+    macroSubstitute(&key,   it->first, ce);
     macroSubstitute(&value, it->second, ce);
+
+    if (key == "")
```

Comment:
Ah, ok. I didn't think about that.
But it is true - if you want to remove a standard header you can add it to the headers list of the template with am empty value.

NTC